File: PDOO/Napakalaki-Ruby/lib/DeathBadConsequence.rb
#encoding: utf-8
require_relative 'BadConsequence.rb'
require_relative 'NumericBadConsequence.rb'
require_relative 'Player.rb'

module NapakalakiGame
  
    class DeathBadConsequence < NumericBadConsequence
      
    def initialize(text)
      super(text,Player.MAXLEVEL,BadConsequence.MAXTREASURES,BadConsequence.MAXTREASURES)
    end
    
  end
end

File: PDOO/Napakalaki-Java/src/NapakalakiGame/DeathBadConsequence.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author Cervi
 */
public class DeathBadConsequence extends NumericBadConsequence{
    
    boolean death;
    
    public DeathBadConsequence(String text){
        super(text, Player.MAXLEVEL, BadConsequence.MAXTREASURES, BadConsequence.MAXTREASURES);
    }
    
    public boolean getDeath(){
        return death;
    }
    
}
